migrations/versions/ba651e8ce9a3_lowercase_discount_code.py

```diff
@@ -0,0 +1,34 @@
+"""lowercase discount code
+
+Revision ID: ba651e8ce9a3
+Revises: 6f7b6fad3f55
+Create Date: 2019-05-21 03:16:20.525011
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'ba651e8ce9a3'
+down_revision = '6f7b6fad3f55'
+
+
+def upgrade():
+    op.execute("UPDATE discount_codes SET code = concat(code, '_') where code not in (SELECT DISTINCT ON (upper(code)) code FROM discount_codes group by event_id, code);",
```

Comment:
Not working correctly. It renamed certain codes which were not in the same event

Reply:
@iamareebjamal ohh, will look into it.